eng/versioning/external_dependencies.txt

```diff
@@ -62,6 +62,7 @@ io.projectreactor.netty:reactor-netty;1.0.4
 io.projectreactor:reactor-core;3.4.3
 io.reactivex:rxjava;1.3.8
 javax.annotation:javax.annotation-api;1.3.2
+javax.json:javax.json-api;1.1
```

Comment:
Please use 1.1.4.
Refs: 
1. https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/external_dependencies.txt#L1
2. https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/scripts/spring_managed_external_dependencies.txt#L461